facebook/react-native
# Fix gradle sync failure with scoped module name

## Motivation

Fix #17029 

I'm using a third react native library named `@0x5e/react-native-alipay`, but Android Studio 3.0 seems not allow project name contains `/`, so I get a gradle sync error:
```
Missing Dependencies
Error:Unable to find module with Gradle path ':@0x5e/react-native-alipay' (needed by module 'app'.)
```

See: https://docs.gradle.org/4.2/release-notes.html#path-separator-characters-in-names-are-deprecated

So can we just replace the `/` to `:` to avoid this error?

## Related PRs

#17000 

## Test Plan

- `react-native init example && cd example`
- `yarn add @0x5e/react-native-alipay`
- `react-native link`
- Open Android Studio 3.0, import `android` folder
- Press `Sync Project with Gradle Files` button
- Gradle Sync should be success

## Release Notes

[CLI] [BUGFIX] [local-cli] - Fix gradle sync error in Android Studio 3.0 with scoped module name
